docs/services/topology.md

```diff
@@ -83,20 +97,44 @@ All changes should be tested on the ITB instance before deploying to production.
 If you can, test them on your local machine first.
 These instructions assume that the code has not been merged to master.
 
-1.  Update the Git clone at `/opt/topology-itb`:
+1.  Update the ITB software installation at `/opt/topology-itb`:
 
         :::console
         # cd /opt/topology-itb
         # git fetch --all
 
-1.  Check out the branch you are testing from:
+1.  Check out the branch you are testing:
 
         :::console
         # git checkout -b <BRANCH>
 
-1.  If the data format has changed in an incompatible way, edit `/etc/opt/topology/config-itb.py`
-    and change `TOPOLOGY_DATA_DIR` and `CONTACT_DATA_DIR` to point to a new directory so the previous data
-    does not get overwritten with incompatible data.
+1.  Verify that you are using the intended data associated with the code you are testing:
+
+    !!! note
+        If the data format has changed in an incompatible way, edit the ITB configuration file and change
```

Comment:
You shouldn't change `CONTACT_DATA_DIR` if the contact data format didn't change, and you shouldn't change `TOPOLOGY_DATA_DIR` if the topology data format didn't change.

Reply:
Also, tell people to save the old version so they can revert the changes later. (This should be common sense but common sense... isn't.)